tensorboard/webapp/widgets/line_chart_v2/lib/coordinator.ts

```diff
@@ -60,6 +61,24 @@ export class Coordinator {
     this.lastUpdated++;
   }
 
+  /**
+   * Returns directionality of the y axis of the output space.
+   *
+   * ↑
+   * | viewBoxYDirection = +1
+   * |
+   * |-------------→
+   * |
+   * | viewBoxYDirection = -1
+   * ↓
+   *
+   * +1: Suitable in 3-D aware context where you have full cartesian coordinates.
+   * -1: Suitable for DOM where <0, 0> is top-left corner of a DOM.
+   */
+  getYAxisDirection(): -1 | 1 {
+    return -1;
+  }
```

Comment:
I really do like the helpful ASCII comment, it's really clear!  An optional alternative is to name this `isYAxisPointedDown(): boolean`, which may help when reading callsites without having to go back to the ascii for reference.